ci-operator/config/terraform-redhat/terraform-provider-rhcs/OWNERS

```diff
@@ -12,6 +12,8 @@ approvers:
 - osherdp
 - sagidayan
 - vkareh
+- xueli181114
```

Comment:
Did you edit manually? 
In the first line of that file, it was written DO NOT EDIT and it autogenerated

Reply:
Recovered. @bardielle